### ISSUE TITLE
A bug in utils.py/is_label.func

### ISSUE THREAD

**Issue by Davidsirui** [keras-team/autokeras#757]
### Bug Description

When testing FeatureEngineering with LGBM backend, we can run the pipeline well with classes number above 2. But if we set the classes number to 2 which means we want a binary classify. It fails. I find that the 0/1 label dind't go through the encode preprocess because the is_label function in utils.py didn't consider it to be a raw label.

### Reproducing Steps
  * Step 1: ...
  * Step 2: ...

### Setup Details
Include the details about the versions of:
 - OS type and version:
 - Python: 
 - autokeras: 
 - scikit-learn:
 - numpy:
 - keras:
 - scipy:
 - tensorflow:
 - pytorch:
